### ISSUE TITLE
adv_post_task watchdog got triggered

### ISSUE THREAD

**Issue by TheSomeMan** [ruuvi/ruuvi.gateway_esp.c#383]
```
W (9715) ADV_POST_TASK: [rx_parse_task] Can't send, MQTT is not connected yet
W (10992) ADV_POST_TASK: [rx_parse_task] Can't send, MQTT is not connected yet
I (11603) ADV_POST_TASK: [adv_post_task] Advertisements in table: 1
I (11603) ADV_POST_TASK: [adv_post_task] i: 0, tag: F4:C6:46:2C:3E:B4, rssi: -37, timestamp: 1633955432:
I (11609) ADV_POST_TASK: [adv_post_task] 0000: 02 01 06 1B FF 99 04 05 14 45 1E 23 C4 9A FF E4 | .........E.#....
I (11619) ADV_POST_TASK: [adv_post_task] 0010: FF E4 03 F8 B0 36 32 74 20 F4 C6 46 2C 3E B4    | .....62t ..F,>.
I (11630) ADV_POST_TASK: [adv_post_task] HTTP POST https://network.ruuvi.com/record: {
        "status":       "online",
        "gw_mac":       "C8:25:2D:8E:9C:2C",
        "timestamp":    "1633955433",
        "nonce":        "1929285348"
}
I (11647) http: [adv_post_task] HTTP POST to URL=https://network.ruuvi.com/record, DATA:
{
        "status":       "online",
        "gw_mac":       "C8:25:2D:8E:9C:2C",
        "timestamp":    "1633955433",
        "nonce":        "1929285348"
}
I (11678) MQTT: [mqtt_task] MQTT_EVENT_BEFORE_CONNECT
E (12751) MQTT_CLIENT: mqtt_message_receive: transport_read() error: errno=0
I (12751) MQTT: [mqtt_task] MQTT_EVENT_ERROR
E (12753) MQTT_CLIENT: esp_mqtt_connect: mqtt_message_receive() returned -1
E (12760) MQTT_CLIENT: MQTT connect failed
I (12768) MQTT: [mqtt_task] MQTT_EVENT_DISCONNECTED
I (12771) LEDS: [mqtt_task] leds_indication_network_no_connection
I (12777) LEDS: [mqtt_task] LED: Start blinking, interval: 200 ms, duty cycle: 50%
I (13037) ruuvi_gateway: [main] free heap: 84748
E (14961) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (14961) task_wdt:  - adv_post_task (CPU 0/1)
E (14961) task_wdt: Tasks currently running:
E (14961) task_wdt: CPU 0: uart_rx_task
Guru Meditation Error: Core  0 panic'ed (Interrupt wdt timeout on CPU0)
```

**Comment by markoaamunkajo** [comment]
[loki.txt](https://github.com/ruuvi/ruuvi.gateway_esp.c/files/7469135/loki.txt)
Task watchdog got triggered. Spotted again in logs on 1.9.0

**Comment by TheSomeMan** [comment]
It's another problem, see #424, fixed in v1.9.1

**Comment by markoaamunkajo** [comment]
Moving to done
